src/boss/cloudvm/local_worker.go

```diff
@@ -28,20 +28,21 @@ func (_ *LocalWorkerPoolPlatform) NewWorker(workerId string) *Worker {
 	}
 }
 
-func (_ *LocalWorkerPoolPlatform) CreateInstance(worker *Worker) {
+func (_ *LocalWorkerPoolPlatform) CreateInstance(worker *Worker) error {
 	log.Printf("Creating new local worker: %s\n", worker.workerId)
 
 	// Initialize the worker directory if it doesn't exist
 	initCmd := exec.Command("./ol", "worker", "init", "-p", worker.workerId, "-i", "ol-min") // TODO fix the "ol-min hardcoding"
 	initCmd.Stderr = os.Stderr
```

Comment:
Writes to stdout/std err should usually be lock-protected so we don't get jumbled output (which is possible when we have two different processes writing to it).  Let's not fix it now, let's just leave a TODO.